PyExpLabSys/common/database_saver.py

```diff
@@ -138,19 +141,20 @@ def add_measurement(self, codename, metadata):
             metadata (dict): The dictionary that holds the information for the
                 measurements table. See :meth:`__init__` for details.
         """
-        DSS_LOG.info('Add measurement codenamed: \'%s\' with metadata: %s', codename, metadata)
-        # Collect column names, values and format strings, a format string is a SQL value
-        # placeholder including processing like e.g: %s or FROM_UNIXTIME(%s)
+        DSS_LOG.info('Add measurement codenamed: \'%s\' with metadata: %s',
+                     codename, metadata)
+        # Collect column names, values and format strings, a format string is a SQL
+        # value placeholder including processing like e.g: %s or FROM_UNIXTIME(%s)
         column_names, values, value_format_strings = [], [], []
         for column_name, value in metadata.items():
             if not COLUMN_NAME.match(column_name):
-                message = 'Invalid column name: \'{}\'. Only column names using a-z, '\
-                          'A-Z, 0-9 and \'_\' and \'$\' are allowed'.format(column_name)
-                raise ValueError(message)
+                msg = 'Invalid column name: \'{}\'. Only column names using a-z, '
+                msg += 'A-Z, 0-9 and \'_\' and \'$\' are allowed'
+                raise ValueError(msg.format(column_name))
 
             if isinstance(value, CustomColumn):
-                # If the value from the metadata dict is a CustomColumn (which is 2 value
-                # tuple), the unpack it
+                # If the value from the metadata dict is a CustomColumn (which
+                # is 2 value tuple), the unpack it
```

Comment:
Not that I'm complaining about having this stuff fixed, but I suggested elsewhere, I would suggest to just up the line limit. 85 chars or what it was seems a old fashioned